src/tenant/initialization.py

```diff
@@ -55,6 +56,12 @@ def create_initial_blocks(user):
     Block.objects.create(block="Default", current_teacher=user)
 
 
+def create_initial_markranges():
+    MarkRange.objects.create(name="A", minimum_mark=85.5, color_light="#FFECB3", color_dark="#FFFF00")
+    MarkRange.objects.create(name="B", minimum_mark=72.5, color_light="#BEFFFA", color_dark="#337AB7")
+    MarkRange.objects.create(name="Pass", minimum_mark=49.5, color_light="#FFB3B3", color_dark="#FF0000")
+
```

Comment:
Please add a simple check to the existintg initialization test that 3 mark ranges have been created in a new deck (see existing test)

Reply:
done